src/Vegetation/BigLeaf/bigleaf_parameterizations.jl

```diff
@@ -0,0 +1,118 @@
+using UnPack # I think I recall someone mentioning we don't need this anymore?
+
+export rubisco_assimilation
+       light_assimilation
+       gross_photosynthesis
+       net_photosynthesis
+       electron_transport
+       ps2_light
+       max_carboxylation
+       max_electron_transport
+       leaf_respiration
```

Comment:
Renato notes that we need faPAR @braghiere

Reply:
Order of things would be: 
(1) a RT code determining fAPAR, 
(2) Ecophysiology model (Farquhar) calculates net carbon assimilation (GPP), 
(3) Stomatal conductance model determines transpiration.